packages/site/src/content/InputDate/InputDateNotes.mdx

```diff
@@ -5,3 +5,14 @@ would be relatively feasible to add in the future:
 
 - allow for the concept of a date-range selection
 - allow for certain days or dates to be excluded from selection
+
+### Web
+
+InputDate pairs with the
+<a href="https://github.com/GetJobber/atlantis/tree/master/packages/components/src/AtlantisContext" target="_blank">AtlantisContext</a>
```

Comment:
to avoid this being a dead end, since we have no documentation or references to this component currently - we'll just link to the publicly available github directory